sw/sw.mk

```diff
@@ -140,7 +140,7 @@ $(foreach link,$(patsubst $(CHS_SW_LD_DIR)/%.ld,%,$(wildcard $(CHS_SW_LD_DIR)/*.
 CHS_CVA6_SDK_IMGS ?= $(addprefix $(CHS_SW_DIR)/deps/cva6-sdk/install64/,fw_payload.bin uImage)
 
 # Create full Linux disk image
-$(CHS_SW_DIR)/boot/linux.gpt.bin: $(CHS_SW_DIR)/boot/zsl.rom.bin $(CHS_SW_DIR)/boot/cheshire.dtb $(CHS_CVA6_SDK_IMGS)
+$(CHS_SW_DIR)/boot/linux-${BOARD}.gpt.bin: $(CHS_SW_DIR)/boot/zsl.rom.bin $(CHS_SW_DIR)/boot/cheshire_$(BOARD).dtb $(CHS_CVA6_SDK_IMGS)
```

Comment:
Yes absolutely thank you!

For the board, this (BOARD)[https://github.com/pulp-platform/cheshire/blob/ck/pr1/cheshire.mk#L15] variable should define more a development board, which is not only about the FPGA development board but also future PCBs.
So maybe the `.dtsi` should stay where it is but the `.dts` can be moved to the `target/fpga`. Just that it will be more annoying for Makefiles. 
Anyways I suppose that a chip instantiating Cheshire will need to re-define this target like I do in [Carfield](https://github.com/pulp-platform/carfield/blob/fpga/linux50mhz/sw/sw.mk#L128)